### PR TITLE
EFEKTA_PWS_MaxPro

### PR DESCRIPTION
**Pull request by smartboxchannel**
add for new device EFEKTA_PWS_MaxPro

### CODE REVIEW

File: devices/custom_devices_diy.js [hunk 1]
```diff
@@ -332,4 +332,25 @@ module.exports = [
         },
         exposes: [e.soil_moisture(), e.battery(), e.illuminance(), e.temperature(), e.humidity()],
     },
+    {
+        zigbeeModel: ['EFEKTA_PWS_MaxPro'],
+        model: 'EFEKTA_PWS_MaxPro',
+        vendor: 'Custom devices (DiY)',
+        description: '[Plant watering sensor EFEKTA PWS Max Pro,  long battery life](http://efektalab.com/PWS_MaxPro)',
```
Comment (Koenkk) [inline devices/custom_devices_diy.js:339]: http://efektalab.com/PWS_MaxPro looks empty

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1155

File: devices/custom_devices_diy.js (hunk 1 above)
Comment (smartboxchannel) [reply devices/custom_devices_diy.js:339]: image

http://efektalab.com/PWS_MaxPro

Comment (Koenkk) [reply devices/custom_devices_diy.js:339]: looks good now!

**Pull request merged**